packages/component-library-react/src/FormFieldTextbox.tsx

```diff
@@ -0,0 +1,146 @@
+import clsx from 'clsx';
+import { ForwardedRef, forwardRef, PropsWithChildren, ReactNode, Ref, useId } from 'react';
+import type { FormFieldProps } from './FormField';
+import { FormField } from './FormField';
+import { FormFieldDescription } from './FormFieldDescription';
+import { FormFieldErrorMessage } from './FormFieldErrorMessage';
+import { FormLabel } from './FormLabel';
+import { Textbox } from './Textbox';
+import { TextboxTypes } from './Textbox';
+
+export type DirType = 'auto' | 'ltr' | 'rtl';
+
+export interface FormFieldTextboxProps extends FormFieldProps {
+  name?: string;
+  invalid?: boolean;
+  disabled?: boolean;
+  label: ReactNode;
+  errorMessage?: ReactNode;
+  description?: ReactNode;
+  status?: ReactNode;
+  readOnly?: boolean;
+  placeholder?: string;
+  min?: number | string;
+  max?: number | string;
+  maxLength?: number;
+  minLength?: number;
+  step?: number | string;
+  autoComplete?: string;
+  pattern?: string;
+  type?: string;
+  required?: boolean;
+  inputRequired?: boolean;
+  inputDir?: string | DirType;
+  value?: string | number;
+  defaultValue?: string;
+  list?: string;
+  size?: number;
+  inputRef?: Ref<HTMLInputElement>;
```

Comment:
Is er een bepaalde reden om hier niet `textboxProps?: TextboxProps` te gebruiken, of de props weg te werken onder een extended interface? Dit lijkt me erg verbose voor onderhoud

Reply:
Ik wil graag React component APIs hebben die ook dezelfde API kunnen hebben voor de web component, dan is een platte structuur fijner.

`extends Pick<TextboxProps, ...>` zou wel handig zijn om onderhoud te verminderen, zou dat een goed compromis zijn?

Reply:
Lijkt mij prachtig!